mmdet3d/ops/sst_modules/sst_ops.py

```diff
@@ -0,0 +1,200 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+
+def make_continuous_inds(win_inds):
+    """Mapping the windows inds to continuous inds.
+
+    The code is modified from original implementation
+    https://github.com/TuSimple/SST
+
+    Args:
+        win_inds (torch.int64): Windows index of each voxel.
+
+    Returns:
+        Tensor: Continuous windows index. The value should
+            be arranged from 0 to the number of unique windows
+            inds.
+    """
+    uni_win_inds, _ = torch.sort(torch.unique(win_inds))
+    num_valid_inds = len(uni_win_inds)
+    max_origin_inds = uni_win_inds.max().item()
+    canvas = -win_inds.new_ones((max_origin_inds + 1, ))
+    canvas[uni_win_inds] = torch.arange(
+        num_valid_inds, dtype=win_inds.dtype, device=win_inds.device)
+    conti_inds = canvas[win_inds]
+    return conti_inds
+
+
+def get_seq_to_win_mapping(batch_win_inds, batching_id_per_voxel,
+                           region_batching_cfg):
+    """Get all transformation info between sequence and batching windows.
+
+    Args:
+        batch_win_inds (Tensor): Windows index of each voxel in a batch.
+            Voxels share a window if they have same index.
+        batching_id_per_voxel (LongTensor): The tensor is the
```

Comment:
May unify the type to torch.Tensor and specifiy its LongTensor property in the description?